### PR TITLE
Python 3 fixes: TACACS, IGMPv3, PPI tests

### PR DESCRIPTION
**Pull request by gpotter2**
This PR adds python 3 compat:
- TACACS
- IGMPv3
- PPI tests
- I noticed that route_add_loopback may override the only working IFace if no LOOPBACK iface detected. Fixed it using dummy guid for ordering + force ether to use "00:00:00:00:00:00" in tacacs building tests, as in that case it won't be overrided

On Windows, route_add_loopback was fixed on https://github.com/secdev/scapy/pull/927. Needed for full python 3 support

### CODE REVIEW

File: scapy/contrib/ppi_geotag.uts [hunk 1]
```diff
@@ -31,6 +31,8 @@ assert GPSTime_Field("GPSTime", None).delta == 0.0
 = Define local_to_utc
 
 def local_to_utc(local_time):
+    if six.PY3:
+        return local_time
```
Comment (gpotter2) [inline scapy/contrib/ppi_geotag.uts]: Seems that this trick (which is a bug under python 2) was fixed on python 3

**Comment by gpotter2** [comment]
@p-l- With https://github.com/secdev/scapy/pull/951, this should get nealy everything working

File: scapy/contrib/ppi_geotag.uts (hunk 1 above)
Comment (p-l-) [reply scapy/contrib/ppi_geotag.uts]: Can you add a comment in the code rather than in the pull-request?

File: scapy/arch/windows/__init__.py [hunk 2]
```diff
@@ -945,7 +950,7 @@ def route_add_loopback(routes=None, ipv6=False, iflist=None):
         if iface.name == scapy.consts.LOOPBACK_NAME:
             IFACES.pop(devname)
     # Inject interface
-    IFACES[data['guid']] = adapter
+    IFACES["{0XX00000-X000-0X0X-X00X-00XXXX000XXX}"] = adapter
```
Comment (p-l-) [inline scapy/arch/windows/__init__.py]: What is this? Why?

Comment (gpotter2) [reply scapy/arch/windows/__init__.py]: data["guid"] needed to be a real one for pcap stuff.
For routing, we don't mind.

Using IFACES[stuff] overwrites the working interface (that we used in data[guid]) with the one with fake routes, making network unreachable.

We only want to add another fake interface that will be used for special loopback routing only

**Comment by gpotter2** [comment]
This should now be mergeable

**Comment by gpotter2** [comment]
@guedou @p-l- Hi ! Could we get this merged ?
I'm willing to finish scapy python 3 integration ASAP

File: scapy/arch/windows/__init__.py [hunk 3]
```diff
@@ -393,8 +393,12 @@ def _check_npcap_requirement(self):
         if not conf.use_npcap:
             raise OSError("This operation requires Npcap.")
         if self.raw80211 is None:
-            dot11adapters = next(iter(_vbs_exec_code("""WScript.Echo CreateObject("WScript.Shell").RegRead("HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Services\\npcap\\Parameters\\Dot11Adapters")""")))
-            self.raw80211 = ("\\Device\\" + self.guid).lower() in dot11adapters.lower()
+            try:
```
Comment (guedou) [inline scapy/arch/windows/__init__.py]: Could you add a comment to explain what this code do ?

File: scapy/contrib/ppi_geotag.uts [hunk 4]
```diff
@@ -31,6 +31,10 @@ assert GPSTime_Field("GPSTime", None).delta == 0.0
 = Define local_to_utc
 
 def local_to_utc(local_time):
+    # BUG workaroung: on python 2, summer time is ignored while converting time to UTC.
```
Comment (guedou) [inline scapy/contrib/ppi_geotag.uts]: s/python/Python/

File: scapy/contrib/tacacs.uts [hunk 5]
```diff
@@ -125,12 +125,12 @@ TacacsAuthorizationReply in pkt and pkt.status == 0  and pkt.arg_cnt is None and
 
 = filled values build -- SSH connection sample secret = test
 
-pkt = Ether()/IP()/TCP(dport=49)/TacacsHeader(seq=2, type=2 , flags=0, session_id=1391595252, length=20)/TacacsAuthorizationReply(status=1, arg_cnt=2, server_msg_len=0, data_len=0, arg_len_list=[11, 1])/TacacsPacketArguments(data='priv-lvl=15')/TacacsPacketArguments(data='a')
+pkt = Ether(src="00:00:00:00:00")/IP()/TCP(dport=49)/TacacsHeader(seq=2, type=2 , flags=0, session_id=1391595252, length=20)/TacacsAuthorizationReply(status=1, arg_cnt=2, server_msg_len=0, data_len=0, arg_len_list=[11, 1])/TacacsPacketArguments(data='priv-lvl=15')/TacacsPacketArguments(data='a')
```
Comment (guedou) [inline scapy/contrib/tacacs.uts]: Why is this needed ? If this is needed, it seems that something was broken along Python 3 porting efforts.
This comment applies to similar modifications.

File: test/regression.uts [hunk 6]
```diff
@@ -5466,10 +5467,11 @@ assert pkt.nscount == 0
 assert pkt.ancount == 0
 
 = Answers - building
-a = UDP()/LLMNRResponse()
-b = UDP()/LLMNRQuery()
+a = UDP()/LLMNRResponse(id=12)
```
Comment (guedou) [inline test/regression.uts]: Is this needed ?

File: scapy/arch/windows/__init__.py (hunk 3 above)
Comment (gpotter2) [reply scapy/arch/windows/__init__.py]: Done !

File: test/regression.uts (hunk 6 above)
Comment (gpotter2) [reply test/regression.uts]: Well this checks that the `answers` works properly with `id` (https://github.com/secdev/scapy/blob/master/scapy/layers/llmnr.py#L46-L49)

Comment (guedou) [reply test/regression.uts]: Thanks !

**Comment by guedou** [comment]
Again, thanks a lot for your work!

**Comment by p-l-** [comment]
Again, thanks @gpotter2! (and thanks @guedou for the review)
